Update asset guide

apaskulin opened the pull request:
<!--- Thanks for submitting a pull request! -->
<!--- If you haven't yet, please read the contributing guide at: -->
<!--- https://github.com/sensu/sensu-docs/blob/master/CONTRIBUTING.md -->
<!--- Provide a general summary of your changes in the Title above. -->

## Description
<!--- Describe your changes in detail -->
- Updates the asset guide to show yml as an example
- Updates to the latest version of PagerDuty asset

To do:
- [x] Get SHA for PagerDuty asset
- [x] Copy to all versions

apaskulin committed:
use yaml

apaskulin committed:
update pagerduty asset

apaskulin committed:
fix json

apaskulin committed:
update definition syntax

apaskulin committed:
copy to all versions

cwjohnston commented on content/sensu-go/5.0/guides/install-check-executables-with-assets.md:
```diff
@@ -11,153 +11,117 @@ menu:
-### Adding an asset to a mutator on creation
 
-[Create an asset][2] (ex. `transformer`), and create a mutator (ex. `transform-metrics`).
+Save the definition to a file, and add to Sensu using sensuctl.
```
```suggestion
Save the definition to a file, e.g. `pagerduty-handler.json`, and add to Sensu using sensuctl.
```

cwjohnston commented on content/sensu-go/5.0/guides/install-check-executables-with-assets.md:
```diff
@@ -11,153 +11,117 @@ menu:
-sensuctl mutator create transform-metrics \
---command "transform --type metrics" \
---runtime-assets transformer
+sensuctl create --file filename.json
```
```suggestion
sensuctl create --file pagerduty-handler.json
```

cwjohnston commented on content/sensu-go/5.0/guides/install-check-executables-with-assets.md:
```diff
@@ -11,153 +11,117 @@ menu:
+To get started with checks, see the [guide to monitoring server resources](../monitor-server-resources).
 
-Once the mutator is setup, it must be attached to a handler to mutate the event output (ex. `influx-db`).
+You can use sensuctl to see available checks, handlers, and other Sensu resources.
```
This feels somewhat disconnected from the rest of what's described here about assets.

Perhaps we move the "to get started with checks" line down to Next steps and get rid of the content about using sensuctl to list available resources?

apaskulin committed:
copy title update to all versions

apaskulin committed:
address review feedback

apaskulin commented:
@cwjohnston Great feedback. Thank you! I've incorporated your changes in https://github.com/sensu/sensu-docs/pull/1309/commits/41e8747c6772f59d4af7a979ce15fa5e669fcc73. I removed the list commands, but kept the reference to the checks guide as part of the third step.

cwjohnston reviewed (approved):
Thanks, looks good!

apaskulin committed:
Merge branch 'master' into asset-yml
